### ISSUE TITLE
Using the right algorithm to detect aircrafts and then optimising it

### ISSUE THREAD

**Issue by FireBoyAJ24** [FireBoyAJ24/Airplane-Type-Detection-System#4]

**Comment by FireBoyAJ24** [comment]
Overfitting
1. We can increase the number of images in each of the planes
2. We can add a pre trained model paramters and weights which can identify planes from other objects
3. We could have a learning rate decay to allow the model to reach a convergence

Underfitting
1. Add more layers to the model
2. Have more epochs
